app/templates/index.html

```diff
@@ -12,6 +12,9 @@
     <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="csrf-token" content="{{ csrf_token() }}" />
+    <script src="/js/util/clipboard.js"></script>
```

Comment:
Can we swap these two lines so that the imports are in lexicographic order?

Reply:
Uh 🤦 I thought I had done that, but I only looked at the names, not the full path… Fixed.